CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ### Added
 
+- Added support for [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/)
+  (NuGet package) traces instrumentation from `4.8.5`.
```

Comment:
if we support only 4.8.5+ this should be documented in config.md. Current version seems to support all available.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3058#issuecomment-1786969711
vs
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3058#issuecomment-1786991155

any preferences?

Reply:
Footer note

Reply:
Then footnote it is - https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3058/commits/3a570e2bd2619fb9a680998819a93a9880c7feda